exercises/GameOfLife/typescript/script.ts

```diff
@@ -0,0 +1,184 @@
+const canvas = document.querySelector<HTMLCanvasElement>("#game");
+const width = canvas.width;
+const height = canvas.height;
+const ctx = canvas.getContext("2d");
+const TILE_SIZE = 20;
+const TILES_X = width / TILE_SIZE;
+const TILES_Y = height / TILE_SIZE;
+ctx.fillStyle = "rgb(100, 240, 150)";
+ctx.strokeStyle = "rgb(90, 90, 90)";
+ctx.lineWidth = 1;
+
+const drawBorders = () => {
+    for(let i=0;i<TILES_X;i++) {
+        ctx.beginPath();
+        ctx.moveTo(i * TILE_SIZE - 0.5, 0);
+        ctx.lineTo(i * TILE_SIZE - 0.5, height);
+        ctx.stroke();
+    }
+for(let j=0;j<TILES_Y;j++) {
+        ctx.beginPath();
+        ctx.moveTo(0, j * TILE_SIZE - 0.5);
+        ctx.lineTo(width, j * TILE_SIZE - 0.5);
+        ctx.stroke();
+    }
+}
+
+const prepareBoard = (): boolean[][] => {
+    const b = [];
+    for(let i=0;i<TILES_X;i++) {
+        const row = [];
+        for(let j=0;j<TILES_Y;j++) {
+            row.push(false);
+        }
+        b.push(row);
+    }
+    return b;
+}
+
+let BOARD = prepareBoard();
+BOARD[1][0] = true;
+BOARD[2][1] = true;
+BOARD[0][2] = true;
+BOARD[1][2] = true;
+BOARD[2][2] = true;
```

Comment:
Probably worth a comment saying this initializes a glider (just so you know when you come back to this in a year and want to know if you can change/delete it)